tidb-cloud/serverless-export.md

```diff
@@ -45,9 +46,9 @@ To export data to Amazon S3, you need to provide the following information:
 - URI: `s3://<bucket-name>/<folder-path>/`
 - One of the following access credentials:
     - [An access key](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html): make sure the access key has the `s3:PutObject` and `s3:ListBucket` permissions.
-    - [A role ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html): make sure the role ARN (Amazon Resource Name) has the `s3:PutObject` and `s3:ListBucket` permissions. 
+    - [A role ARN](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html): make sure the role ARN (Amazon Resource Name) has the `s3:PutObject` and `s3:ListBucket` permissions. Note that only clusters with AWS cloud provider support role ARN.
 
-For more information, see [Configure External Storage Access for TiDB Cloud Serverless](/tidb-cloud/serverless-external-storage.md#configure-amazon-s3-access).
+For more information, see [Configure TiDB Cloud Serverless External Storage Access](/tidb-cloud/serverless-external-storage.md#configure-amazon-s3-access).
```

Comment:
```suggestion
For more information, see [Configure External Storage Access for TiDB Cloud Serverless](/tidb-cloud/serverless-external-storage.md#configure-amazon-s3-access).
```

Reply:
reason: use the same title as that in https://docs.pingcap.com/tidbcloud/serverless-external-storage/